.ci/update-sources.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+set -e
+
+source ./.ci/util.sh
+
+if [[ -z $1 ]]; then
+  echo "version is not set"
+  echo "Usage: outside-sources.sh <version>"
```

Comment:
Let's make name of file
`update-releasenotesxml.sh`

Reply:
Done